customizations/views.py

```diff
@@ -134,76 +183,52 @@ class ViewCreateCustomization(LoginRequiredMixin, FormView):
-                pdf_ticket_attach = request.POST.get('pdf_ticket_attach')
-                template = get_object_or_404(
-                    BaseTicketTemplate,
-                    pk=select_design_template)
-                ticket = TicketTemplate.objects.create(
-                    select_design_template=template,
-                    message_ticket=message_ticket,
-                    show_event_sequence=show_event_sequence,
-                    show_ticket_type_sequence=show_ticket_type_sequence,
-                    hide_ticket_type_price=hide_ticket_type_price,
-                    footer_description=footer_description,
-                    double_ticket=double_ticket,
-                )
-                custom_email = CustomEmail.objects.create(
-                    message=message,
-                    logo=links['dropbox'],
-                    logo_local=links['local'],
-                    logo_path=links['path'],
-                    logo_name=links['name'],
-                    logo_url=links['dropbox'],
-                    image_partner=partner_links['dropbox'],
-                    image_partner_local=partner_links['local'],
-                    image_partner_path=partner_links['path'],
-                    image_partner_name=partner_links['name'],
-                    image_partner_url=partner_links['dropbox'],
-                )
-                Customization.objects.create(
-                    user=self.request.user,
-                    ticket_template=ticket,
-                    custom_email=custom_email,
-                    name=name,
-                    pdf_ticket_attach=pdf_ticket_attach,
-                )
-                if not UserWebhook.objects.filter(user=request.user).exists():
-                    token = get_token(request.user)
-                    webhook_id = create_webhook(token)
-                    UserWebhook.objects.create(
-                        user=self.request.user,
-                        webhook_id=webhook_id,
+                if compose_customization(request):
+                    return HttpResponseRedirect('/')
+                else:
+                    form.error = 'An unexpected error has occurred. ' \
+                                 'Please try again.'
+                    return render(request, self.template_name, {
+                        'form': form, 'modal_crop_image': modal_crop_image}
                     )
-                return HttpResponseRedirect('/')
             else:
+                form.error = 'The form is not valid. Please check that ' \
+                             'you have completed the required ' \
+                             'fields correctly.'
                 return render(request, self.template_name, {
-                    'form': form}
+                    'form': form, 'modal_crop_image': modal_crop_image}
                 )
 
 
 def update_customization(request, pk):
-
+    t = loader.get_template('includes/modal_crop_image.html')
```

Comment:
In general we avoid to use this sort of `t` variable names, it does not have any semantic value. If you say instead `template` for examples, it would be more readable.